server/api-manager.js

```diff
@@ -932,16 +938,14 @@ export default class MindbodyQueries {
     return false;
   }
 }
+
 const MindbodyAccess = new MindbodyQueries();
 export default MindbodyAccess;
 
 /*example code
 var client = new mindbodyQueries();
 client.getAuth()
-  .then((value) => {
-    client.AUTH_TOKEN = value.AccessToken;
-    return client.getClients();
-  })
+  .getClients()
   .then((value) => {
     console.log(value);
   })
```

Comment:
This example code is something that we could improve for learning purposes even though most of the code for retrieving and using MindBody data is already written.

Here is some example code that me and Daniel came up with before.

/*example code:
import MindbodyAccess from "../../api-manager.js";
//...
MindbodyAccess.getAuth()
.then(() => {
return MindbodyAccess.getClients();
})
.then((clientsList) => {
console.log(clientsList);
})
.catch((error) => {
console.log("Error occurred in [name the context]: " + error.toString());
});
*/